src/components/DocContainer.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import styled from 'styled-components/native';
+import DocHolder, { Missing, Submitted } from 'components/DocHolder';
+import { TextBold } from 'assets/fonts/Fonts';
+import { Colors } from 'assets/Colors';
+
+
+interface Props { 
+    caseType: string;
+    uploadStatus: boolean; //based on all submissions
+}
+
+
+const Container = styled.View`
+    border-radius: 8px;
+    width: 349px;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+    border: ${Colors.lightGray};
+    padding-left: 19px;
+    padding-right: 22px;
+    
+`
+
+const Header = styled.View`
+    height: 29px;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    margin: 24px;
+
+
+`
+
+const docs = ['First DACA Application', 'Employment Authorization', '2020 Tax Return', 'Employment Document']
+
+
+const DocContainer = (props: Props) => { 
+    return (
+        <Container>
+            <Header>
+            {props.uploadStatus ? Submitted : Missing}
+            <TextBold>{props.caseType}</TextBold>    
+            </Header>
+            {docs.map(name => <DocHolder title={name} submitted={ true }/>)}
```

Comment:
React requires that you add a unique "key" prop for each listed item! See [here](https://www.akashmittal.com/each-child-list-unique-key-prop/) for explanation.

Reply:
The submitted props should take in props.submitted since you're already passing `true` down from the Documents screen!